lang/cs/Org.Apache.REEF.Wake/packages.config

```diff
@@ -18,6 +18,10 @@ specific language governing permissions and limitations
 under the License.
 -->
 <packages>
+  <package id="Microsoft.Avro.Core" version="0.1.0" targetFramework="net451" developmentDependency="true" />
+  <package id="Microsoft.Avro.Tools" version="0.1.0" targetFramework="net451" developmentDependency="true" />
+  <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net451" developmentDependency="true" />
```

Comment:
If this is not for .NET project, it should use the same version as non .Net project,